src/configuration/configuration-base.ts

```diff
@@ -21,12 +23,16 @@ const DEBUG_LOGGER = debug('testcafe:configuration');
 
 export default class Configuration {
     protected _options: Dictionary<Option>;
-    protected readonly _filePath: string | null;
+    protected _filePath: string | null;
+    protected readonly _filePaths: (string | null)[];
```

Comment:
I guess it would be better to make it an optional array of strings. Why would we need to allow null as an element?
```
protected readonly _filePaths?: string[];
```

Reply:
Also, I feel that `filePaths` is not a descriptive name. What's about `defaultPaths`?